mne/cov.py

```diff
@@ -1431,11 +1435,12 @@ def _get_whitener_data(info, noise_cov, picks, diag=False, rank=None,
         noise_cov = cp.deepcopy(noise_cov)
         noise_cov['data'] = np.diag(np.diag(noise_cov['data']))
 
-    scalings_ = dict(mag=1e12, grad=1e11, eeg=1e5)
+    _scalings = DEFAULTS['scalings']
```

Comment:
Are you sure that the whitening and rank values should be the same?  Every DOC string regarding rank estimation (and whitening, for that matter) says that dict(eeg=1e6, grad=1e13, mag=1e15) will be used.  But, the two places that had different values from this were in <code>prepare_noise_cov</code> and <code>_get_whitener_data</data>.